config/routes.rb

```diff
@@ -7,6 +7,7 @@
   get 'web', to: 'web#index'
   get 'web/data', :defaults => { :format => 'js' }
   get 'web/show', to: 'web#show'
+  get 'web/about', to: 'web#about'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.